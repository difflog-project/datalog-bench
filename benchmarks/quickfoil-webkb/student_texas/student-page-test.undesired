1774
817
2244
2265
2175
2000
2089
2144
915
2083
1834
2185
1746
2055
2073
1941
2018
1823
2236
663
2171
1709
2029
639
1788
2270
2145
1854
2229
2113
1710
2135
2188
1953
1804
1998
1792
1814
1883
669
818
1863
1777
1778
1745
2016
2178
2047
2060
2151
1918
1899
2116
2214
2152
1886
2134
917
1972
2049
1967
2063
1781
2027
2044
680
809
1798
1977
2002
1865
1770
2156
2269
2098
901
892
1891
2209
2025
1907
1739
2023
2220
1817
681
2170
2207
2187
2148
2096
2017
1726
2099
1841
2048
1744
823
2094
1828
668
1896
2146
1731
2013
2200
1867
1789
1923
1991
2079
1728
1824
1898
1940
1978
1810
2080
641
2224
2205
1708
1809
2091
2030
1773
1786
890
1962
2123
1821
2222
1784
2194
2212
642
815
1990
2112
640
1807
2109
918
1779
2210
573
1721
899
2133
1925
662
2129
2126
1922
2069
1919
1840
2271
2124
650
2260
1881
2012
2215
913
1857
2082
1800
1924
1876
2110
1912
2218
2142
2041
2010
1983
1749
2238
2084
2206
1771
2074
2173
2256
1748
660
1965
2064
2176
1838
1960
2075
2020
1811
1772
902
1750
1851
2090
1895
1911
1819
1994
1822
2213
1951
1847
1751
1813
675
1874
1725
1737
2263
651
2153
1878
2182
2102
2227
1947
925
895
1982
1879
2183
2086
2042
1856
1963
1931
1787
1738
1884
2036
682
1862
2150
2093
1917
2248
1936
654
2189
2004
909
819
2211
1812
1837
807
666
1915
920
2199
1767
897
2273
1888
649
1806
571
2015
2232
1782
916
677
672
2028
1839
1909
1959
2225
2191
2240
2067
1766
1833
1961
2193
1872
2006
2119
1741
2217
1987
1921
808
1734
658
1992
1805
2085
2077
2019
2111
2121
2032
2208
1914
2254
1894
1718
1916
1858
1785
2197
1826
2261
2066
2164
891
1995
2159
657
2181
1970
1713
1893
2051
647
1831
1729
644
2235
1756
2241
1946
1815
2223
1765
2105
1957
2072
1712
1852
1988
1889
2058
821
648
813
2160
673
2095
1958
2034
665
1829
2108
1890
912
2103
2158
2136
1714
683
2258
2253
2177
2242
2280
2279
1776
1723
656
2038
2165
1794
2266
1864
2166
1902
2274
2043
1966
2155
1842
2120
894
2267
2186
2097
2252
2125
2076
2008
1910
1943
1973
2059
1717
1877
1759
1830
816
2087
2237
1975
1780
2033
1974
2115
1932
1897
1832
1905
1808
1948
1996
2139
1730
1885
2168
2003
1735
812
1986
2088
2007
2065
1843
2106
2117
1954
2179
814
2131
2040
2138
2219
2045
1956
2230
2118
824
2239
1880
2140
1801
2272
908
1762
1753
2149
820
2226
679
2130
2107
1732
643
1733
2247
1908
2201
1855
572
1999
1796
924
1844
2061
1904
904
2251
1827
1719
2009
1997
911
2021
674
2228
2070
2249
1930
1790
2011
1711
2132
2203
810
1900
1882
1873
903
1920
1845
2056
1795
2141
2202
926
1752
1906
822
1799
2037
1803
758
896
1933
1720
2180
1945
645
2005
2255
1755
1949
1743
1820
1887
2221
1783
2022
1913
2198
2195
638
2143
2162
1952
2054
1791
2128
921
1950
1870
1860
1971
1985
1866
655
1926
2233
1980
2154
2246
922
2174
1768
1849
2264
2276
2250
907
671
653
2163
2169
1868
1853
1937
2092
2100
2277
1715
2031
906
1758
2192
1955
1793
1944
2137
1859
2053
1761
2071
1981
1716
1903
2278
1769
1869
2052
1928
664
2050
2046
1740
1929
1742
2190
661
2014
1836
1736
1747
2257
1927
914
900
1901
1724
1802
1861
1993
923
2231
1757
1775
1934
1764
905
670
2035
2167
1942
2216
1979
2104
1969
2057
2196
1825
1816
2234
893
2147
811
1964
678
1797
919
1848
2172
2039
2062
1850
2184
2101
889
1871
2161
1984
1760
1968
2078
652
1938
659
898
2275
1846
1835
2127
2122
1939
2026
1763
2157
1722
2204
2081
1989
2259
2243
676
667
1892
2024
910
2245
2114
1818
1976
1754
2268
2262
1935
646
2068
1875
2001
1727
