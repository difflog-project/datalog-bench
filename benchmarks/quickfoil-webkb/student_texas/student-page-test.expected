314
372
339
301
414
327
321
371
386
320
357
395
429
422
326
410
396
369
426
407
427
418
323
373
416
399
313
356
325
352
343
341
324
383
361
365
393
387
329
417
298
367
296
299
394
364
406
415
297
319
403
391
401
400
381
412
309
348
405
370
304
342
404
377
333
308
347
420
289
392
306
397
303
307
305
295
428
375
408
346
424
290
351
421
318
338
402
360
353
384
312
334
315
300
409
378
358
317
355
345
283
368
379
349
363
385
285
302
362
328
322
292
359
330
284
413
376
336
335
286
344
311
293
310
332
388
288
366
425
411
382
380
389
390
282
354
350
291
331
294
287
316
423
374
398
337
340
419
